cmd/theatre-envconsul/main.go

```diff
@@ -133,6 +133,32 @@ func mainError(ctx context.Context, command string) (err error) {
 			}
 		}
 
+		var filePaths = environment{}
+		var pathTBD = "PATH_TBD"
```

Comment:
Does this make more sense as a const? Or can we use a more idiomatic sentinel value such as empty string?

Reply:
I think an empty string would work!